Trakt/Extensions.cs

```diff
@@ -121,7 +123,40 @@ public static bool MetadataIsDifferent(this TraktMovieCollected collectedMovie,
                || collectedMovie.Metadata.Resolution != resolution
                || collectedMovie.Metadata.Is3D != is3D
                || collectedMovie.Metadata.Hdr != hdr
-               || collectedMovie.Metadata.MediaType != null;
+               || collectedMovie.Metadata.MediaType != TraktMediaType.digital;
+    }
+
+    /// <summary>
+    /// Checks if metadata of new collected episode is different from the already collected.
+    /// </summary>
+    /// <param name="collectedEpisode">The <see cref="TraktEpisodeCollected"/>.</param>
+    /// <param name="episode">The <see cref="Episode"/>.</param>
+    /// <returns><see cref="bool"/> indicating if the new episode has different metadata to the already collected.</returns>
+    public static bool MetadataIsDifferent(this TraktEpisodeCollected collectedEpisode, Episode episode)
+    {
+        var mediaStreams = episode.GetMediaStreams();
+        var defaultVideoStream = mediaStreams.FirstOrDefault(x => x.Index == episode.DefaultVideoStreamIndex);
+        var audioStream = mediaStreams.FirstOrDefault(x => x.Type == MediaStreamType.Audio);
+
+        var resolution = defaultVideoStream.GetResolution();
```

Comment:
This will crash if defaultvideostream is null. If it cannot be null, use `First()`